### PR TITLE
Update PipelineResource details view to match common layout

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Add tabs for details / YAML on the PipelineResource details
page to match the common layout being introduced for all
resource types.

https://github.com/tektoncd/dashboard/issues/969

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/PipelineResource/PipelineResource.js [hunk 1]
```diff
@@ -138,129 +142,199 @@ export /* istanbul ignore next */ class PipelineResourceContainer extends Compon
...
+                  />
+                </p>
+                <p>
+                  <span>
+                    {intl.formatMessage({
+                      id: 'dashboard.metadata.labels',
+                      defaultMessage: 'Labels:'
+                    })}
+                  </span>
+                  {formattedLabelsToRender.length === 0
```
Comment (steveodonovan) [inline src/containers/PipelineResource/PipelineResource.js]: Is the equals 0 check needed here ?

File: src/containers/PipelineResource/PipelineResource.js [hunk 2]
```diff
@@ -138,129 +142,199 @@ export /* istanbul ignore next */ class PipelineResourceContainer extends Compon
...
-                      </TableHeader>
-                    ))}
-                  </TableRow>
-                </TableHead>
-                <TableBody>
-                  {rows.map(row => (
-                    <TableRow {...getRowProps({ row })}>
-                      {row.cells.map(cell => (
-                        <TableCell key={cell.id}>{cell.value}</TableCell>
+        <Tabs selected={0}>
```
Comment (steveodonovan) [inline src/containers/PipelineResource/PipelineResource.js]: Just a default value ? What does this prop do ?

File: src/containers/PipelineResource/PipelineResource.js (hunk 1 above)
Comment (AlanGreene) [reply src/containers/PipelineResource/PipelineResource.js]: unfortunately yes, otherwise we end up rendering a '0' in the UI

Comment (AlanGreene) [reply src/containers/PipelineResource/PipelineResource.js]: hrmm I guess not

File: src/containers/PipelineResource/PipelineResource.js (hunk 2 above)
Comment (AlanGreene) [reply src/containers/PipelineResource/PipelineResource.js]: selects the first tab by default so its content is rendered on load
after it's mounted Carbon manages the selected tab via internal state and this prop no longer applies

File: src/containers/PipelineResource/PipelineResource.js (hunk 1 above)
Comment (AlanGreene) [reply src/containers/PipelineResource/PipelineResource.js]: updated

**Comment by AlanGreene** [comment]
> Didn't do a full comparison of each field but I assume no new values for the translatable strings needed here ?

That's right, using the existing standard strings for the details / YAML tabs to ensure consistency across all types.

**Pull request merged**